### ISSUE TITLE
Nether star alternative

### ISSUE THREAD

**Issue by squeek502** [squeek502/VeganOption#1]
No real ideas for this yet, but it should be quite involved (and not in the "complicated/expensive crafting" sense).

EDIT: Quote from a later post of mine:

> Here's some guidelines for what I see as an ideal implementation:
> - Nether stars are dangerous to get, and can lead to the player's death
> - The mechanic for obtaining nether stars is interactive. That is, the player needs to react correctly--in real time--to things that are happening
> - The mechanic does not get stale. That is, it should never get boring or feel repetitive

**Comment by BarbasTheDog** [comment]
Maybe a formation of quartz and gold blocks (trying to imitate the colours of a nether star) with a puddle of lava at the centre. Then you thow a diamond block (or something else that's quite valuable) into the puddle, a cool sound effect happens, the whole thing blows up a bit (so it's semi-hard to automate) and boom! nether star.

just something that came to mind, thought I might share.

I love the mod btw. I played while a while as vegan myself (not extending my peaceful ways to the undead though :P )

I really approve of the mod :D

**Comment by squeek502** [comment]
Glad you like it. :smiley: 

If possible, I'd like to avoid a straight-forward crafting-type thing and instead shoot for something more interactive/dynamic; like some sort of mini-game.

As for what that entails, I'm not totally sure, but I'd like to try to keep the "difficult thing you have to defeat" aspect of the wither fight while removing the "kill the wither" aspect.

**Comment by freyquency** [comment]
Perhaps crafting a wither with blank heads (or a new kind of head) unleashes something that gives a nether star, but doesn't want to tear apart the world and kill everything?

**Comment by BBoldt** [comment]
I have some ideas.
- Must be in the nether
- Make some sort of ritual construct sort of thing, maybe similar to the Nether Reactor from minecraft pocket edition: http://minecraft.gamepedia.com/Nether_Reactor
- This construct, when activated, spawns a sort of (maybe randomly generated) dungeon with unbreakable walls and traps. Mazes, piston traps, things you need to shoot to activate, etc. I'm thinking something similar to a puzzle that would be in a tomb raider game.
- At the end of the dungeon you get to take the nether star from some sort of pedestal, and then GET OUT QUICK because the dungeon collapses.

So basically, I'm thinking it should be something adventurous and slightly dangerous that may or may not resemble stuff from minecraft pocket edition, tomb raider, or indiana jones. Something you have to beat both with mind and skill.

**Comment by reteo** [comment]
I recall seeing something in which you had the three wither heads and the four soul sands in the crafting grid in the correct configuration for a wither boss (instead of being placed), as well as a diamond sword and bow in the remaining bottom two corners.

**Comment by ghost** [comment]
I recall a peaceful-only recipe in which you placed three wither skulls on the top row of the crafting grid, three soulsand on the middle row, and a diamond sword, soulsand, and bow on the bottom row.  Perhaps you can simply modify this recipe to work regardless of the game mode?

**Comment by BBoldt** [comment]
I think that is the peaceful table from extra utilities.

Also, in my opinion, that's a pretty lame solution.

**Comment by ghost** [comment]
Maybe a glowstone block in the center, diamond blocks in the top, bottom, left, and right, and four enchantment books in the corners?

Hmmm, perhaps use a two-phase recipe, skipping the enchantment books for the first recipe, to form an unstable nether star, which needs to be crafted with an enchantment book within ten seconds (like unstable ingots from ExU) to become a stable Nether Star... or else you lose the star completely?

Or maybe you take heat damage from an unstable nether star (light from glowstone amplified and focused by all that diamond) until a fire resistance enchantment (or potion) is crafted with it?

**Comment by ghost** [comment]
> Note that I don't really like the idea of it being craftable, no matter how complicated the crafting recipe is.

So... these are out:
- Crafting a nether star is out.
- Killing or harvesting one from living creatures goes against the nature of this mod.
- Spawning a nether star from breaking a vanilla nether block is likely out, since there are very few non-trivial options available for such breaking.

But these aren't:
- Making a device that can spawn a nether star could be interesting.  Perhaps combining overworld and underworld materials into something really nuts.  But does that count as crafting?
- Assembling multiple blocks (vanilla and crafted) into a makeshift device (similar to the way raw ender is generated).

Am I missing anything?  Enchanting a block of diamond with a custom enchantment, perhaps?

Or can part of it be crafted?  In the same vein as the Ender Pearl?  Making the Nether star container out of glowstone and diamond blocks, and then somehow generating its essence through some device or multiblock structure?

Perhaps use the recipe I gave above to make an empty nether star container.  Then, using eight quartz blocks surrounding a diamond block, make a stellar core.  This core needs to be surrounded in all 6 sides with a glowstone blocks for a specified period of time for the essence to be ready.  Once it's ready, the stellar core becomes raw stellar essence, ready to be crafted with the nether star container.

Eh.  I'm not really sure what else to suggest.

**Comment by squeek502** [comment]
@reteo, yeah you're on the right track. I realize that I haven't been very specific about this. I have a vague idea in my mind of the type of thing that I'd want to make, but I don't really know what exactly it should be.

Here's some guidelines for what I see as an ideal implementation:
- Nether stars are dangerous to get, and can lead to the player's death
- The mechanic for obtaining nether stars is interactive. That is, the player needs to react correctly--in real time--to things that are happening
- The mechanic does not get stale. That is, it should never get boring or feel repetitive

As an example, here's an idea I just had that might work:
- The player creates some structure or performs some ritual that 'spawns' a nether star in the world a long distance above them (has to be done in a large clearing)
- Once the nether star is spawned, blocks/entities start raining down from the sky that deal damage when they hit the player. Once they're on the ground, however, they act as normal blocks and no longer deal damage
- These randomly falling blocks will stack on top of eachother over time, and your goal is to 'ride' them up to where the nether star is and collect it

This would potentially satisfy all of the preceding guidelines. Could also enforce a few rules to stop it from being easily exploitable (can't leave the radius without the nether star despawning, can't fly within the radius, etc).

Let me know what you think of something like that, or if you have any ideas for similar types of things (basically, a dangerous mini-game).

**Comment by ghost** [comment]
So, in other words, you're aiming for something like the Division Sigil, but for generating a star?  Before it can be used, it has to be activated at midnight by killing a creature near an enchantment table, surrounded with redstone trails with a full view of the sky?

Why not the opposite?  A worktable with the previously-mentioned empty nether star, also surrounded by redstone trails, but the ritual is at noon, and _you yourself_ have to take damage in order for a chance (not a guarantee) of the star becoming real.

**Comment by squeek502** [comment]
> So, in other words, you're aiming for something like the Division Sigil, but for generating a star?

Not necessarily. The method of starting it isn't as important as the mini-game element of it. 

I think it'd actually be better to design it in reverse--create a fun and interesting mini-game first, and then figure out how to make it fit within the Minecraft world/lore. And what I posted above is just a random idea that popped into my head; there are an infinite number of other possibilities that might be better.

**Comment by jena-th** [comment]
An idea, close to the "Howl's Moving Castle":
perform a ritual during a non-raining night: that start a "star shower" and very rarely a star fall spawning a small and very fast sprite which you have to touch to get a nether star before it despawn.

Why "killing a wither" ?
- Soulsand is cheap.
- wither skeletton heads are just a question of grinding, even less with beheading TiCon tools.
- There is few ways to kill it safely.

Why being vegan, and not kill a wither to get a star ?
3 points:
- The ritual should cost, but not a ludicrous amount of resources, because the soulsand and the heads are cheap. So, make it cheaper.
- The star is falling randomly, so that's a first mini-game: will it fall and where ?
- Next you have the small "hit and run" with the sprite, during the night, it's not out of danger, and it's not 100% sure to get him before he go back to the sky.
  Overall it would be cheaper but non farm-able and still fun and even beautiful !

**Comment by ghost** [comment]
> Soulsand is cheap.
> wither skeletton heads are just a question of grinding, even less with beheading TiCon tools.

You do realize that wither skulls are cheap, too.  One papier mache and 8 black inks makes a wither skull... and it works.

**Comment by jena-th** [comment]
I think this recipe should go away

**Comment by squeek502** [comment]
@kane-thornwyrd, not a bad idea, although it seems like it might be a little too dependent on random factors (how dangerous it is would vary a ton depending on mods included/settings/randomness).

See https://github.com/squeek502/VeganOption/issues/16 for mob head balancing discussion. I agree that the ease of crafting wither skulls is a problem.

**Comment by jena-th** [comment]
@squeek502 you're not able to level your mod depending on the X modpacks that use it, vanilla would be easy, blood'n'bone-like nearly sure-death, you can add a settings to tweak the speed of the sprite, if he take pauses, how long he run, how far. Modpack creators would have to adjust those settings, that's all.

**Comment by ghost** [comment]
I know you don't want a crafting recipe, but Wither's can be so easy to kill/grind with other mods, maybe it should be possible. This pack is a great alternative to mob drop crops and other peaceful solutions; I see no reason to make it harder for players. Something that requires compressing nether resources around wither heads could require a fair amount of harvesting, silk touch, exploration, etc. which would be it's own exploration and reward; a scavenger hunt and quest. Make players climb to the top of the nether and dig to the bottom of lava lakes, skulk through nether fortresses and gather the essence of the nether. Maybe include some integration, so some infrastructure from other mods would be used (if they're installed). The creation of alloys and specially bred nether-crops (if using agricraft) would be a type less reflex oriented minigame and push the star to later in the game.

**Comment by Magnogen** [comment]
Sculk, you say?

I couldn't resist. Sorry. Personally I've never gotten a nether star, or a beacon for that matter - so I'm not sure if my opinion is valuable on this subject, but I do have some ideas.

I like the idea of exploring fortresses, bastions, and other areas in the nether to find shards of a nether star, or perhaps component parts of these - adding to the grind that I know a lot of players face to get a wither.

Summoning something via a ritual sparked something in me though - perhaps instead of fighting a wither-like beast, it could be more of a natural process. Something akin to actually trying to contain a sun - playing on the nether _star_ theme. Players could summon said star underwater or in an ice spikes biome to assist with cooling it down, and making it slightly easier to deal with, all while still staying true to the vegan mantra. It could still work for a wither drop, with the implication that this incredible power source was used to power the thing.

'Till next time!

**Comment by DancingDragoness** [comment]
What if you could cure the wither? Like what if you treat it as an act of necromancy and you must free the souls bound to it and they gift you the nether star that they were bound to as thanks? Perhaps with a few wither rose seeds as well???

**Comment by elifoster** [comment]
Interesting concept. Do you have any idea as to what that would look like in gameplay?

**Comment by DancingDragoness** [comment]
> Interesting concept. Do you have any idea as to what that would look like in gameplay?

Okay, apologies for the late response. Life became super busy. I’m genuinely honored by your interest in the idea. As for gameplay, after giving it some thought, here’s what I have in mind for what that might look like.

Historically, while exorcisms and similar rites never follow a single specific format, a recurring theme across many traditions is smoke and the burning of certain materials such as oils, incense, and particular plants such as garlic or hyssop.

Perhaps the player must trap the Wither above a campfire so it remains in contact with the smoke, and then add certain herbs to the fire such as alliums (as a stand-in for garlic) and hyssop (perhaps with a chance to drop from grass in meadows) using left click. Adding these plants would alter the smoke for a limited time, indicated by a subtle visual change, I'm thinking in color and perhaps in shape as well. And the wither would show the same particles as zombie villagers being cured to indicate that it's actually working. The plants might need to be used in a specific order, turning the process into a procedural ritual rather than a single action. Regardless, the player would need to keep feeding the fire as needed to maintain the smoke’s effectiveness. 

During this time, I imagine the wither still being able to hit the player or perhaps the curing process causes a harmful area-based effect around the wither the player must be in to add the plants to the fire.  Alternatively, it could be that the wither's essence or something is temporarily being moved to the player as it is being forced from the wither and the player, being a living soul, cannot be taken over by the essence and thus it dissipates, but sustains much damage from it.

Progress could be indicated via the Wither’s health bar, or via a separate bar if you prefer. Instead of a normal death, the Wither would split into three spirits resembling ghostly villagers. At that point, the Nether Star could drop (possibly alongside wither rose seeds, if desired) either directly after defeat or as something the spirits gift via tossing before vanishing, having been set free.
